### PR TITLE
Added default value to avoid UNBOUND VARIABLE error when using -p or -u

### PR DESCRIPTION
**Pull request by adduarte**

### CODE REVIEW

File: bzira [hunk 1]
```diff
@@ -317,9 +317,9 @@ jira_workspace="${JIRA_WORKSPACE:-$default_jira_workspace}"
 jira_project="${JIRA_PROJECT:-$default_jira_project}"
 jira_board="${JIRA_BOARD:-$default_jira_board}"
 # shellcheck disable=SC2153
```
Comment (pierreprinetti) [inline bzira]: neat!
you can probably get rid of those two `# shellcheck disable=SC2153` now

**Review by pierreprinetti — CHANGES_REQUESTED** [review]

**Comment by pierreprinetti** [comment]
What is the difference between these two? I thought you had to add the column, but your code works without it

```bash
"${var:-x}" # column
```

and

```bash
"${var-x}" # no column
```

File: bzira (hunk 1 above)
Comment (adduarte) [reply bzira]: done

**Comment by pierreprinetti** [comment]
Did you mean to commit `test.sh`?

**Comment by adduarte** [comment]
> Did you mean to commit `test.sh`?

No. removed.

**Review by pierreprinetti — APPROVED** [review]
Thanks!

**Pull request merged**